scripts/create-latest-json.sh

```
@@ -0,0 +1,264 @@
+  fi
+fi
+}
+
+prepare_directory_files() {
+  local artifact_path=$1
+  local platform_key=$2
+
+  # Check if the directory already exists
+  if [[ -d "$BASE_DIR/$platform_key" ]]; then
+    echo "Directory $BASE_DIR/$platform_key already exists. Removing it..."
+    rm -rf "$BASE_DIR/$platform_key"
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)